Tests/StreamChatTests/StreamChatStressTests/Cached_StressTests.swift

```diff
@@ -53,16 +53,23 @@ final class Cached_Tests: StressTestCase {
 
     /// This test doesn't really check the results, it just checks the program doesn't crash.
     func test_cachedWorksCorrentlyInMulthithreadedEnvironment() {
+        let queues = [
```

Comment:
Checking if not using global queues makes it more stable since global queues might be busy with something else as well.

Reply:
I will rerun the unit-test step on CI a couple of time for seeing if it helps. First run was good.

Reply:
Trying something else as well.